components/molecules/Experiment.js

```diff
@@ -13,7 +13,7 @@ export const Experiment = (props) => {
   };
   return (
     <a
-      className={`shadow-experiment-shadow p-4 border-b-4 h-400px ${
+      className={`shadow-experiment-shadow p-4 border-b-4 min-h-400px ${
```

Comment:
Is this because text is falling outside the card @nibivi77 ?

Reply:
I want to clarify because Xiaopu wants text that doesn't fit in the card to be cut off and replaced with a "...". Similar to the CDS blog here when a description exceeds the size of the card:

https://digital.canada.ca/blog/

Reply:
Right now, we have this on 320px:
![image](https://user-images.githubusercontent.com/77116011/125514872-13ac93cd-dcda-4913-a6bb-5ffe2da99129.png)

With that `min-h-400px` it gives this:
![image](https://user-images.githubusercontent.com/77116011/125515077-7259ce08-8334-48ce-b9a7-74f3f51644b0.png)

Reply:
An alternative is to have the box stay 400px and have it scroll-able if the text is longer than the box.

Reply:
The issue with the `overflow-ellipsis` is that it will cut out some info. and if we want to expand it, it will enlarge the card regardless.